### ISSUE TITLE
A11y: Video element cookie accept button has no aria-label

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#6408]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

----

*_Actual behaviour:_*

Vimeo/Youtube cookie accept buttonhas no aria-label

*_Expected behaviour:_*

Vimeo/Youtube cookie accept button should be marked up with aria-label

*_How to reproduce:_*

1. Create a Vimeo/Youtube iframe video via shopping experiences
2. Toggle Load Video after confirmation from user
3. Do not accept cookies
4. Inspect the accept button

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40413**



[created from NEXT-40413, comment 535723]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40413, comment 535838]

**Comment by shopwareBot** [comment]
[public] Thank you so much for submitting this. In this case, an aria label isn’t necessary as the button is a native element, written in semantic HTML: it is already keyboard and screen reader compatible.

 Per the official guide to ARIA, this is always the preferred solution: [https://www.w3.org/TR/using-aria/#NOTES](https://www.w3.org/TR/using-aria/#NOTES) 

 Again, thank you for supporting our goal to make Shopware more accessible.



[created from NEXT-40413, comment 536820]
